nvflare/fuel/f3/stats_pool.py

```diff
@@ -117,3 +157,68 @@ def from_dict(cls, d: dict):
             else:
                 raise ValueError(f"invalid pool type {t}")
             cls.pools[k] = p
+
+    @classmethod
+    def dump_summary(cls, file_name: str):
+        stats_dict = cls.to_dict()
+        json_string = json.dumps(stats_dict, indent=4)
+        with open(file_name, "w") as f:
+            f.write(json_string)
+
+    @classmethod
+    def dump_records(cls, csv_file_name: str):
+        recs_dict = cls.get_records()
```

Comment:
It makes sense to dump_summary at the end of the RUN. But for dump_records(), should we dump / write the record every time when we get a record, instead of appending / accumulating them, then do a huge dump at the end?